Test/Integration/Search/SearchTest.php

```diff
@@ -8,18 +8,34 @@
 
 class SearchTest extends TestCase
 {
-    public function testSearch()
+    /** @var Product */
+    protected $productIndexer;
+
+    /** @var Data */
+    protected $helper;
+
+    public function setUp(): void
     {
-        /** @var Product $indexer */
-        $indexer = $this->getObjectManager()->create(Product::class);
-        $indexer->executeFull();
+        parent::setUp();
 
+        $this->productIndexer = $this->objectManager->get(Product::class);
+        $this->helper = $this->getObjectManager()->create(Data::class);
+
+        $this->productIndexer->executeFull();
         $this->algoliaHelper->waitLastTask();
+    }
 
-        /** @var Data $helper */
-        $helper = $this->getObjectManager()->create(Data::class);
-        list($results, $totalHits, $facetsFromAlgolia) = $helper->getSearchResult('', 1);
+    public function testSearch()
+    {
+        list($results, $totalHits, $facetsFromAlgolia) = $this->helper->getSearchResult('', 1);
```

Comment:
Let's broaden our test cases a little to actually verify that expected products, skus, and colors are found against certain assertion values and not just that there are results.